language/bosnian/date_lang.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * CodeIgniter
+ *
+ * An open source application development framework for PHP
+ *
+ * This content is released under the MIT License (MIT)
+ *
+ * Copyright (c) 2014 - 2019, British Columbia Institute of Technology
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @package	CodeIgniter
+ * @author	EllisLab Dev Team
+ * @copyright	Copyright (c) 2008 - 2014, EllisLab, Inc. (https://ellislab.com/)
+ * @copyright	Copyright (c) 2014 - 2019, British Columbia Institute of Technology (https://bcit.ca/)
+ * @license	https://opensource.org/licenses/MIT	MIT License
+ * @link	https://codeigniter.com
+ * @since	Version 1.0.0
+ * @filesource
+ */
+defined('BASEPATH') OR exit('Nije dozvoljen direktan pristup');
+
+$lang['date_year'] = 'Godina';
+$lang['date_years'] = 'Godine';
+$lang['date_month'] = 'Mjesec';
+$lang['date_months'] = 'Mjeseci';
+$lang['date_week'] = 'Nedjelja';
+$lang['date_weeks'] = 'Nedjelje';
+$lang['date_day'] = 'Dan';
+$lang['date_days'] = 'Dani';
+$lang['date_hour'] = 'Sat';
+$lang['date_hours'] = 'Sati';
+$lang['date_minute'] = 'Minut';
+$lang['date_minutes'] = 'Minute';
+$lang['date_second'] = 'Sekunda';
+$lang['date_seconds'] = 'Sekunde';
+
+$lang['UM12']	= '(UTC -12:00) Baker/Howland Island';
```

Comment:
This section doesn't seem to be translated. It's true that I see one another example as well, in another language, where this is not translated...

Reply:
I made a translation
[date_lang.zip](https://github.com/bcit-ci/codeigniter3-translations/files/6046875/date_lang.zip)

Reply:
[date_lang.zip](https://github.com/bcit-ci/codeigniter3-translations/files/6046743/date_lang.zip)
 made a translation

Reply:
@jekkos please do it. thanks!

Reply:
@gxgpet Translations were added